components/molecule/field/src/index.js

```diff
@@ -123,7 +124,12 @@ const MoleculeField = ({
           />
         </div>
       )}
-      <div className={CLASS_INPUT_CONTAINER}>
+      <div
+        className={cx(
+          CLASS_INPUT_CONTAINER,
+          withBox && `${CLASS_INPUT_CONTAINER}--with-box`
+        )}
```

Comment:
This class does not respect suit naming
[https://suitcss.github.io/](https://suitcss.github.io/)

Reply:
Yeah, i don't know how to name it correctly ):  This was the better name i came up with. Any suggestion? May be, just --box?

Reply:
Changed to: --aligned! 👍